### PR TITLE
tc-report documentation

### PR DESCRIPTION
**Pull request by petertgiles**
Documentation on the tc-report directory.

### CODE REVIEW

File: documentation/tc-report.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+### Step 2 - Apply your changes to the `_site` branch
+
+The [`/GCTC-NTGC/tc-report`](https://github.com/GCTC-NTGC/tc-report) repository maintains a `_site` branch which contains only the generated files from the `_site` directory.  This provides an isolated view of __only the generated files__.
+```
+# Change into your cloned tc-report directory (wherever you have it)
+cd ~/repos/tc-report
+
+# Make (and checkout) a staging branch:
+git checkout -b staging-branch origin/main
```
Comment (petertgiles) [inline documentation/tc-report.md:31]: Honestly, I'm not sure what purpose the staging-branch has or what would happen if this step was skipped.

File: documentation/tc-report.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+This section details the workflow for applying updates to the files in the `tc-report` directory.
+
+### Step 1 - Apply the changes in the source repository
+
+Follow the regular github process to apply the changes in the [`/GCTC-NTGC/tc-report`](https://github.com/GCTC-NTGC/tc-report) repository: 
+1. Clone the repository to your workstation
+2. Make a feature branch with your changes
+3. Open a pull request
+4. Have your feature branch merged into `main`.
```
Comment (mnigh) [inline documentation/tc-report.md:19]:
```suggestion
4. Have your feature branch merged into `main`
```

**Review by mnigh — CHANGES_REQUESTED** [review]
a trivial change requested for punctuating (or not) a list consistently but otherwise, very nice!

**Review by mnigh — APPROVED** [review]
👍

**Review by yonikid15 — APPROVED** [review]
This looks great Peter 👍

**Pull request merged**